code/modules/cargo/packs/engineering.dm

```diff
@@ -214,6 +214,14 @@
 	contains = list(/obj/item/circuitboard/computer/sat_control)
 	crate_name= "shield control board crate"
 
+/datum/supply_pack/engineering/ceturtlenecks
```

Comment:
But why? Can't this be added the garments bag for the chief engineer?

Reply:
Because oranges said so